system-test/error-reporting.ts

```diff
@@ -49,8 +49,8 @@ const envKeys = [
 class InstancedEnv {
   injectedEnv: {[key: string]: {}};
   _originalEnv: {[key: string]: {}};
-  apiKey: string;
-  projectId: string;
+  apiKey!: string;
```

Comment:
This seems like a good opportunity to remove usage of the `assign` library instead of masking it with `!`. What do you think?

Reply:
At least for this PR, I like having the smallest change necessary.  A larger refactor would be better for a follow-up PR.